notebook/static/notebook/js/menubar.js

```diff
@@ -87,6 +88,7 @@ define([
         }
     };
 
+    // deprecate ?
```

Comment:
It does not do anything anymore  beyond triggering an event. 
Wondering if we should just get rid of the extra layer of abstraction, but we should check if it is used.